app/src/main/java/org/mozilla/fenix/components/searchengine/FenixSearchEngineProvider.kt

```diff
@@ -70,6 +73,10 @@ class FenixSearchEngineProvider(
         )
     }
 
+    fun defaultEngineIdentifiers() = runBlocking {
+        defaultEngines.await().list.map { it.identifier }
```

Comment:
Nit: `defaultEngines` is confusing, rename it to `bundledEngines` or `installedEngines`, and then `BUNDLED_SEARCH_ENGINES` renamed as `EXTRA_SEARCH_ENGINES` or something? I can see this getting confusing with the "default search engine" which is used for the default search everywhere else.